pkgs/by-name/cl/cloak-pt/package.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}: let
+  version = "2.9.0";
+in
+  buildGoModule {
+    pname = "Cloak";
+    inherit version;
+
+    src = fetchFromGitHub {
+      owner = "cbeuw";
+      repo = "Cloak";
+      rev = "v${version}";
+      hash = "sha256-IclSnSJAUSWWAk8UZbUJLMVcnoZk5Yvsd1n3u67cM2g=";
+    };
+
+    vendorHash = "sha256-kkb/gPnDbJvfc5Qqc5HuM1c9OwOu1ijfO7nNNnY3mOo=";
+
+    doCheck = false;
+
+    ldflags = [
+      "-X main.version=master(6b08af0)"
+    ];
+    meta = with lib; {
+      homepage = "https://github.com/cbeuw/Cloak/";
+      description = "Pluggable transport that enhances traditional proxy tools like OpenVPN to evade sophisticated censorship and data discrimination.";
```

```suggestion
      description = "Pluggable transport that enhances traditional proxy tools like OpenVPN to evade sophisticated censorship and data discrimination";
```
description should not end with punctuation marks

---

also don't forget the commit where you add yourself to the maintainers